tide_automated_listing.install

```diff
@@ -13,7 +13,9 @@ use Drupal\Core\Serialization\Yaml;
 function tide_automated_listing_update_8002() {
   $paragraphs = \Drupal::entityQuery('paragraph')
     ->condition('type', 'automated_card_listing')
-    ->exists('field_paragraph_auto_listing')
+    ->exists('field_paragraph_auto_listingconfig
```

Comment:
@asarfara cc @anthony-malkoun what is this?

Reply:
Typo